src/main/java/ch/uzh/ifi/hase/soprafs24/service/TaskService.java

```diff
@@ -28,30 +26,23 @@ public class TaskService {
 
```

Comment:
The UserService is leveraged for method calls regarding the User entity. Other unused repositories have been removed.